event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepository.java

```diff
@@ -50,7 +57,23 @@
     static final String SQL_DISTINCT_STREAM_ID = "SELECT DISTINCT stream_id FROM event_log";
     private static final String FAILED_TO_READ_STREAM = "Failed to read stream {}";
     private static final String STREAM_ID = "STREAM_ID";
-    private static final String SQL_PAGE = "SELECT * FROM event_log WHERE stream_id=? ORDER BY sequence_id ASC LIMIT ? OFFSET ?;";
+
+    //HEAD - Latest
```

Comment:
Remove comments

Rename the constants and method names so that they are in keeping with the names used in LINK enum.

Constants should be SQL_GET_HEAD, SQL_GET_NEXT etc

Reply:
Done